backtrace-library/src/main/cpp/backtrace-native.cpp

```diff
@@ -306,8 +310,16 @@ Java_backtraceio_library_BacktraceDatabase_addAttribute(JNIEnv *env, jobject thi
 }
 
 JNIEXPORT void JNICALL
-Java_backtraceio_library_base_BacktraceBase_dumpWithoutCrash(JNIEnv *env, jobject thiz,
-                                                             jstring message) {
-    DumpWithoutCrash(message);
+Java_backtraceio_library_base_BacktraceBase_dumpWithoutCrash__Ljava_lang_String_2(JNIEnv *env,
```

Comment:
Why did this function name change even though arguments are the same? Maybe we don't need to care about this

Reply:
Because we extended the current function to prevent API break.

Reply:
Ah okay sounds good. Thank you!